packages/assets-controllers/src/MultichainBalancesController/MultichainBalancesController.test.ts

```diff
@@ -426,4 +440,20 @@ describe('BalancesController', () => {
       mockBalanceResult,
     );
   });
+
+  it('resumes updating balances after unlocking KeyringController', async () => {
+    const { controller, messenger } = setupController();
+
+    messenger.publish('KeyringController:lock');
+
+    await controller.updateBalance(mockBtcAccount.id);
+    expect(controller.state.balances[mockBtcAccount.id]).toBeUndefined();
```

Comment:
For the record: If for whatever reason the `setupController` updates the balances (since we're mocking `isUnlocked = true`, then we might really have a defined balance for that `mockBtcAccount.id` and the `KeyringController:lock` won't clear the balance (which is expected).

I believe updating the balances asynchronously in the ctor is probably not ideal and we could revisit this some day to use a `async init()` method instead.